### PR TITLE
Adding weighted_skip parameter to MutilayerPerceptron layer

### PR DESCRIPTION
**Pull request by riya-singh28**
## Description

By Adding weighted_skip parameter to MutilayerPerceptron layer, it can be used for skip connection as well. Hence, it can be used in place of ResidualBlock for Global and Local Message Passing in MXMNet Model.

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [x] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [x] Run `mypy -p deepchem` and check no errors
  - [x] Run `flake8 <modified file> --count` and check no errors
  - [x] Run `python -m doctest <modified file>` and check no errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by vsomnath — CHANGES_REQUESTED** [review]

File: deepchem/models/tests/test_layers.py [hunk 1]
```diff
@@ -703,6 +703,30 @@ def test_MultilayerPerceptron_overfit():
     assert np.allclose(output, y, atol=1e-2)
 
 
+@pytest.mark.torch
+def test_weighted_skip_MultilayerPerceptron():
```
[the remark this answers is not in the document]
Comment (riya-singh28) [reply deepchem/models/tests/test_layers.py:707]: Yea sure, I will.

**Review by ARY2260 — APPROVED** [review]
LGTM

**Comment by vsomnath** [comment]
LGTM

**Review by vsomnath — APPROVED** [review]

**Review by rbharath — APPROVED** [review]
LGTM

**Pull request merged**